fit/extract_rawyield.py

```diff
@@ -96,6 +96,18 @@ def fit(config_file): # pylint: disable=too-many-locals,too-many-statements
 
     with open(cfg["cutset_file_name"], "r") as yml_cfg:  # pylint: disable=unspecified-encoding
         cut_set = yaml.load(yml_cfg, yaml.FullLoader)
+    
+    particle = cfg["particle"]
+    if particle == "Bplus":
+        pdgId = 521
+        decayChannel = r"\overline{D}^{0} \pi^{\plus}"
```

Comment:
Ciao @apalasciano!
Thanks a lot for your work, it looks nice to me! I would consider using `snake_case` format for variables, as we plan to add a linter (pylint) to the repository in the future, but since it is not there yet you can do it in a next iteration if you prefer

```suggestion
        decay_channel = r"\overline{D}^{0} \pi^{\plus}"
```